jupyter_fsspec/handlers.py

```diff
@@ -251,6 +251,77 @@ def post(self):
         self.finish()
 
 
+# ====================================================================================
+# Handle Syncing Local and Remote filesystems
+# ====================================================================================
+class FilesystemSyncHandler(BaseFileSystemHandler):
+    def initialize(self, fs_manager):
+        self.fs_manager = fs_manager
+
+    async def get(self):
```

Comment:
`get` implies no change of state. I would call both methods "post", but have them on different endpoints or with different operation parameters as with the other filesystem commands.